feedback.md

```diff
@@ -7,17 +7,20 @@
 |   Answered comprehension questions 	|   	|
 |   Both Teammates contributed to the codebase	|   	|
 |   Regular Commits with meaningful commit messages 	|   	|
-|  **Wave 1**	|   	|
-|  `Scoring` class	|   	|
+|  **Scoring class**	|   	|
 |  `score` method	|   	|
-|   Uses hash to store the alpha score	|   	|
+|   Uses appropriate data structure to store the alpha score	|   	|
 |   Tests `score` edge cases (one-letter word, 7 letter word)	|   	|
 |  `highest_scoring_word` method	|   	|
-|   Appropriately handling edge cases	|   	|
+|   Appropriately handles edge cases	|   	|
 |   Tests `highest_scoring_word` edge cases (ties, no words, no array)	|   	|
-|  **Wave 2**	|   	|
-|  `Player` class	|   	|
-|	**Baseline**	|	|
-|  Readable code with consistent indentation. 	|   	|
-|   **Extras**	|   	|
-
+|  **Player class**	|   	|
+|  Uses an array to keep track of words played	|   	|
+|  Uses existing code to create `highest_scoring_word` and `highest_word_score`	|   	|
+|  Returns `true` or `false` from `won?` method |   |
+|  Tests edge cases on the `play(word)` method for words that cannot be scored |     |
+|  **TileBag class**	|   	|
+|  Uses a data structure to keep the set of default tiles in the bag	|   	|
```

Comment:
Should we be more explicit in the project requirements that they keep the default tiles info in a data structure (versus having code that pushes n copies of a letter into each new tile bag)?